dnsapi/dns_simply.sh

```diff
@@ -204,6 +204,9 @@ _simply_add_record() {
     return 1
   fi
 
+  _info "Waiting 60 seconds for DNS changes to be written"
+  _sleep 60
```

Comment:
why do we need to sleep 60 here?
If it's for the dns record to propagate,  we should ask the user to use `--dnssleep 60` command line parameter to wait.


Imagine you have 10 domains in one cert,   if you sleep here,  you will sleep 10 minutes.   
but if you use "--dnssleep 60",  you will just have to sleep 60 seconds.

Reply:
You are absolutely right. I must admit I just adopted the code from the original contributer of PR #3673 without much review.

I've removed the unnecessary sleep now.